alacritty/src/display/hint.rs

```diff
@@ -117,6 +117,8 @@ impl HintState {
             },
             // Cancel hint highlighting on ESC.
             '\x1b' => self.stop(),
+            // Do the same but with Ctrl+c
+            '\x3' => self.stop(),
```

Comment:
These two statements can be combined using `|`, see the lines just above this. The comment should then also be updated to say `ESC/Ctrl+C` of course.

Also the same as the backspace key (`\x08`), the hexadecimal needs to be padded with a `0`.